tests/foreman/api/test_contentview.py

```diff
@@ -168,13 +147,11 @@ def test_cv_associate_custom_content(self):
         yum_repo = entities.Repository(product=product).create()
         yum_repo.sync()
         content_view = entities.ContentView(organization=org).create()
-        self.assertEqual(len(content_view.read_json()['repositories']), 0)
+        self.assertEqual(len(content_view.read().repository), 0)
```

Comment:
The create does not return the repository field? If yes then this can be changed to `self.assertEqual(len(content_view.repository), 0)`